### DISCUSSION TITLE
Question about discussions and issues.

### DISCUSSION THREAD

**Discussion by pietru2004** [pocketbase/pocketbase#6127] [Q&A] [answered]
If I can ask, How many AI/LLM generated discussions or issues is there need to be dealt with every day?

**Comment by ganigeorgiev** [comment, marked as answer]
It is not much but it is enough to be annoying (_anywhere between 0-50 per month_). So far the spam/AI submitted content could be grouped in:
- malicious accounts that posts links to pirated "software" that it is most likely malware (I try to delete and block these immediately and report the account to the GitHub support with excerpt of the submitted content; GitHub usually ban them within a day or two)
- bot accounts of some AI services that reply to existing issues with comments and/or create PRs for issues no one asked them for (I usually delete, or hide if I'm not sure that it is bot, and eventually report them to the GitHub support for "Disruptive content" if it continues)
- real users that post AI generated comments/summaries that often doesn't make sense (these I either hide as "Off-topic" or delete directly and if after a warning or two the user continue to do it, I just block them)
- real users that posts AI generated code and asking why it doesn't work (this is probably the most frustrating for me and it usually results in closing the discussion and asking the user to read the docs and stop using LLM, e.g. https://github.com/pocketbase/pocketbase/discussions/5989)

If the reason for your question is because you have received GitHub notification for a reply to some of your older posts and you no longer can find it, it is because either I've deleted it (_or GitHub has deleted their entire account_) due to the reasons above or because the user has deleted it on their own.

Note that I have no control over the user notifications and if you don't want to receive notifications for your posts you'll have to either manually "unsubscribe" from the sidebar for each specific post OR update your GitHub profile settings.

**Reply by pietru2004** [reply]
I did see it occur once...
